tests/test_fixtures/rule_based_profiler/example_notebooks/SelfInitializingExpectations.ipynb

```diff
@@ -684,7 +684,7 @@
     "        \"quantile_statistic_interpolation_method\": \"nearest\",\n",
     "        \"quantile_bias_correction\": False,\n",
     "        \"quantile_bias_std_error_ratio_threshold\": None,\n",
-    "        \"estimator\": \"bootstrap\",\n",
+    "        \"estimator\": \"exact\",\n",
```

Comment:
@NathanFarmer Let us not make this change.

Reply:
✅